src/hardware/adlib.cpp

```diff
@@ -889,28 +972,37 @@ Module::Module(Section *configuration)
 	const auto read_from = std::bind(&Module::PortRead, this, _1, _2);
 	const auto write_to = std::bind(&Module::PortWrite, this, _1, _2, _3);
 
-	// 0x388 range
+	// 0x388-0x39b ports (read/write)
 	constexpr io_port_t port_0x388 = 0x388;
 	WriteHandler[0].Install(port_0x388, write_to, io_width_t::byte, 4);
 	ReadHandler[0].Install(port_0x388, read_from, io_width_t::byte, 4);
-	// 0x220 range
+
+	// 0x220-0x224 ports (read/write)
 	if (!single) {
 		WriteHandler[1].Install(base, write_to, io_width_t::byte, 4);
 		ReadHandler[1].Install(base, read_from, io_width_t::byte, 4);
 	}
-	//0x228 range
+	// 0x228-0x229 ports (read/write)
 	WriteHandler[2].Install(base + 8u, write_to, io_width_t::byte, 2);
+
+	// 0x228 port (read)
 	ReadHandler[2].Install(base + 8u, read_from, io_width_t::byte, 1);
```

Comment:
comment above mentions:
`0x228-0x229 ports (read/write)`

But the read handler only installs `read_from` on port 0x228 and not 0x229.

(does Gold allow reading from 229h as well?)

Reply:
No, the code is correct, and my explanatory comments are apparently doing more harm than good 🤣 Must have been tired when I wrote this... Fixed.